### PR TITLE
Make Title Block Dates Translatable

### PR DESCRIPTION
**Pull request by AWearring**
Updated Published and Updated Date text to be translatable - should have remembered to do this on https://github.com/localgovdrupal/localgov_publications/pull/154

### CODE REVIEW

**Comment by ekes** [comment]
This is possibly a nit-pick. But maybe someone can find the guidance on this. I think translation should either be:
```
<span>{{ 'Published'|t }}: </span>{{ published_date }}
```
But actually more likely
```
{% trans %}
<span>Published:</span> {{ published_date }}
{% endtrans %}
```
to make any of the possible variations of translation possible.

**Comment by finnlewis** [comment]
@AWearring thanks for this, any comment in reply to comment from @ekes above?

**Comment by AWearring** [comment]
@finnlewis Probably should be `<span>{{ 'Published'|t }}: </span>{{ published_date }}` taking out the colon makes sense. 
This will need to be updated in a lot of places, I suspect

**Comment by finnlewis** [comment]
Discussing with Merge Tuesday crew, general thinking is that it would be good to follow the suggestion by @ekes above.

```
{% trans %}
<span>Published:</span> {{ published_date }}
{% endtrans %}
```

This will allow support for other languages that might want to re-order the structure of the sentence.

**Comment by finnlewis** [comment]
Note that the translator will then translate the entire string: `<span>Published:</span> {{ published_date }}`

**Comment by AWearring** [comment]
@finnlewis @ekes Updated to use `{% trans %}` as discussed at Merge Tuesday

File: templates/localgov-publication-page-header-block.html.twig [hunk 1]
```diff
@@ -32,11 +32,19 @@
...
+  </div>
+{% endif %}
 
-  {% if last_updated_date %}
-    <div class="lgd-page-title-block__updated"><span>Last updated: </span>{{ last_updated_date }}</div>
-  {% endif %}
+{% if last_updated_date %}
+  <div class="lgd-page-title-block__updated">
+    {% trans %}
+      <span>{{ 'Last updated: '|t }}</span>{{ last_updated_date }}
```
Comment (markconroy) [inline templates/localgov-publication-page-header-block.html.twig]: You don't need the `|t` filter here, since it's inside a `{% trans %}` tag.

**Review by markconroy — CHANGES_REQUESTED** [review]

File: templates/localgov-publication-page-header-block.html.twig (hunk 1 above)
Comment (AWearring) [reply templates/localgov-publication-page-header-block.html.twig]: Removed :-)

**Review by markconroy — APPROVED** [review]

**Comment by rupertj** [comment]
Merged. Thanks!

**Pull request merged**